build/repo.targets

```diff
@@ -1,9 +1,11 @@
 <Project>
   <ItemGroup>
     <!-- Optimal ThreadCount for current hardware -->
-    <Scenarios Include="-n Plaintext --kestrelThreadCount 2" />
-    
+    <Scenarios Include="-n Plaintext --kestrelTransport Libuv --kestrelThreadCount 2" />
```

Comment:
Before setting kestrelThreadCount implied libuv. That is no longer the case. Made libuv explicit here.